packages/agents/sequencer/src/lib/operations/auctions.ts

```diff
@@ -146,7 +146,6 @@ export const executeAuctions = async (_requestContext: RequestContext) => {
   // Handling each domain in parallel, but each individual transfer synchronously. This is to account
   // for the fact that one transfer's auction might affect another. For instance, a router might have
   // 100 tokens to LP, but bid on 2 100-token transfers. We shouldn't send both of those bids.
-  // RS: going to try out parallelizing it anyways, its okay if a few of them fail
```

Comment:
This is worth discussing imo!! This could be parallelized, within reason - if we suspect that the majority of routers' bids won't get picked anyway (due to randomness).

Tradeoff is : routers polluting the network with overbidding vs routers underbidding conservatively. Arguably the sequencer should manage which routers don't have sufficient funds and filter them out... but if we pile all the work on the sequencer to do this, it won't scale very well